config/Migrations/20231108194647_AddInstructorNotificationsPerEvent.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddInstructorNotificationsPerEvent extends AbstractMigration
+{
+    public function up()
+    {
+        $this->table('events')
+            ->addColumn('notifyInstructorRegistrations', 'boolean', [
+                'default' => true,
+                'limit' => null,
+                'null' => false,
+            ])
+            ->addColumn('notifyInstructorCancellations', 'boolean', [
+                'default' => true,
+                'limit' => null,
+                'null' => false,
+            ])
```

Comment:
Actually thinking about it now, I probably should default this to false so that members are unexpectedly getting notifications for events they've already created...

Reply:
I would default to false.  Some people don't care about it and/or want emails.  Those that do can check the box(es).